llvm/test/CodeGen/AArch64/trunc-to-tbl.ll

```diff
@@ -706,7 +706,7 @@ define void @trunc_v11i64_to_v11i8_in_loop(ptr %A, ptr %dst) {
 ; CHECK-NEXT:  LBB6_1: ; %loop
```

Comment:
Changes to big-endian tests look suspicious (not properly checked them yet)

Reply:
Seems okay after using NVCAST, though there is a slight regression here (due to the lack of post-incremented store now).